skywalking-collector/skywalking-collector-worker/src/main/java/com/a/eye/skywalking/collector/worker/tools/EsClient.java

```diff
@@ -0,0 +1,30 @@
+package com.a.eye.skywalking.collector.worker.tools;
```

Comment:
Strange package name. It is not a tool. It is a storage component or storage implementations.